render-app/src/test/java/org/janelia/alignment/spec/TileBoundsRTreeTest.java

```diff
@@ -220,4 +240,45 @@ private List<TileBounds> buildListForZ(final double z) {
 
         return list;
     }
+
+    @SuppressWarnings("SameParameterValue")
+    private List<OrderedCanvasIdPair> buildExpectedPairs(final double pZ,
+                                                         final int pTileIndex,
+                                                         final double qZ,
+                                                         final int... qTileIndexes) {
```

Comment:
This is super specific, but: when referring to the mathematical meaning, the plural of index is indices. In contrast, the plural form indexes is used when referring to databases or documents.

Reply:
Ahh - I concede defeat on this one 🥲